index.js

```diff
@@ -0,0 +1,185 @@
+// INDEX
+// - Variables
+// - Initialize
+// - Navigation
+// - Step 1
+// - Step 2
+// - Step 3
+
+
+
+const step0 = document.getElementById('step0');
+const step1 = document.getElementById('step1');
+const step2 = document.getElementById('step2');
+const step3 = document.getElementById('step3');
+const tax = 0.08;
+
+let customerBankAccount = 0;
+let customerSpendLimit = 0;
+let phoneChoice = "";
+let accessoryChoice = "";
+let bundleSize = "";
+let totalSpent = 0;
+let costPerBundle = 0;
+
+
+
+initialize();
+
+// *** each step gets its own display component for input and/or info ***
+
+function initialize() {
+  toggleDisplaySteps(step0, step1); //TODO - optional argument not working? had to add 2nd arg
```

Comment:
The net approach here is still imperative - ideally you would call `showStepOne()` right here, and `showStepOne` would take care of calling `toggleDisplaySteps`. Reading through the code I can then go straight from here to `showStepOne`, and follow along with the execution logic. Instead, I currently have to scroll to find `toggleDisplaySteps`, see that it changes some css classes, and then its a dead end, not clear what happens next.

Reply:
I switched `showStepZero` in there instead of just running that toggle by itself.

I ended up adding a `Step 0` that serves as the "press begin" bit to explain what you're actually using.